x-pack/plugins/alerting/server/routes/rule/apis/bulk_edit/bulk_edit_rules_route.ts

```diff
@@ -39,15 +40,19 @@ const buildBulkEditRulesRoute = ({ licenseState, path, router }: BuildBulkEditRu
       router.handleLegacyErrors(
         verifyAccessAndContext(licenseState, async function (context, req, res) {
           const rulesClient = (await context.alerting).getRulesClient();
-          const bulkEditData: BulkEditRulesRequestBodyV1 = req.body;
+          const actionsClient = (await context.actions).getActionsClient();
 
+          const bulkEditData: BulkEditRulesRequestBodyV1 = req.body;
           const { filter, operations, ids } = bulkEditData;
 
           try {
             const bulkEditResults = await rulesClient.bulkEdit<RuleParamsV1>({
               filter,
               ids,
-              operations,
+              operations: transformOperationsV1({
```

Comment:
Now that we do not add the `type` to the actions, do we still need the `transformOperationsV1` util?

Reply:
I will keep it, it is not doing that much!